src/topology/continuous_function/algebra.lean

```diff
@@ -811,35 +812,39 @@ We now provide formulas for `f ⊓ g` and `f ⊔ g`, where `f g : C(α, β)`,
 in terms of `continuous_map.abs`.
 -/
 
-section
-variables {R : Type*} [linear_ordered_field R]
+namespace continuous_map
 
--- TODO:
--- This lemma (and the next) could go all the way back in `algebra.order.field`,
--- except that it is tedious to prove without tactics.
--- Rather than stranding it at some intermediate location,
--- it's here, immediately prior to the point of use.
-lemma min_eq_half_add_sub_abs_sub {x y : R} : min x y = 2⁻¹ * (x + y - |x - y|) :=
-by cases le_total x y with h h; field_simp [h, abs_of_nonneg, abs_of_nonpos, mul_two]; abel
+section lattice
+variables {α : Type*} [topological_space α]
+variables {β : Type*} [topological_space β]
 
-lemma max_eq_half_add_add_abs_sub {x y : R} : max x y = 2⁻¹ * (x + y + |x - y|) :=
-by cases le_total x y with h h; field_simp [h, abs_of_nonneg, abs_of_nonpos, mul_two]; abel
+/- C(α, β) is a lattice ordered group -/
 
+@[to_additive continuous_map.has_le.le.add_covariant_class]
+instance [lattice β] [comm_group β] [topological_group β] [covariant_class β β (*) (≤)] :
+  covariant_class C(α, β) C(α, β) (*) (≤) :=
+begin
+    refine ⟨λ f g₁ g₂ hg₁₂, _⟩,
+    intro x,
+    simp only [to_fun_eq_coe, mul_apply, mul_le_mul_iff_left],
+    apply hg₁₂,
 end
```

Comment:
Does this work?

```suggestion
⟨λ f g₁ g₂ hg₁₂ x, begin
    simp only [to_fun_eq_coe, mul_apply, mul_le_mul_iff_left],
    apply hg₁₂,
end⟩
```

I would imagine you can golf even further to a single line, though I'm not at Lean right now to try.

Reply:
Golfed